### ISSUE TITLE
Bug: No Skip to content feature 

### ISSUE THREAD

**Issue by Ekwuno** [surrealdb/www.surrealdb.com#312]
### Describe the bug

Currently focus rings are trapped within lists that aren't visible to the user. This makes navigation harder when tabbing through. 

There also is no skip to main content button to take users to main content and avoid tabbing through multiple elements 

### Steps to reproduce

Tab through the website  a good place to see this is the documentation route. 

### Expected behaviour

Users should be able to exit or enter lists as needed and also skip to main content 

### SurrealDB version

SurrealDB 1.0

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by akankshadharkar** [comment]
After a sync last week - the issue is not happening everywhere on the website anymore (possibly post the website redesign?) but there are _some_ edge cases. 
Will update the issue description accordingly.

**Comment by Ekwuno** [comment]
We still need to add a skip to main content feature. Will update the issue accordingly

**Comment by Ekwuno** [comment]
This is already implemented in the docs migration so I will close this now.
